test/e2e/adminUI/tests/group002Home/uxTestHomeView.js

```diff
@@ -2,107 +2,121 @@ var adminUI = require('../../adminUI');
+		browser.initialForm = browser.page.initialForm();
+		browser.listScreen = browser.page.list();
+		browser.deleteConfirmation = browser.page.deleteConfirmation();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinScreen.signin();
+		browser.app.waitForElementVisible('@homeScreen');
 	},
 	after: function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.allView.logoutIconLink)
-			.pause(browser.globals.defaultPauseTimeout)
-			.end();
+		browser.app.signout();
+		browser.end();
 	},
-	'Home view should allow clicking a nav menu item such as Access to show the list of items': function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.allView.accessMenu)
-			.waitForElementVisible(adminUI.cssSelector.listView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.allView.fieldListsMenu)
-			.waitForElementVisible(adminUI.cssSelector.listView.id)
-			.pause(browser.globals.defaultPauseTimeout);
+	'Home view should allow clicking a nav menu item such as Access and Fields to show the list of items': function (browser) {
+		browser.app
+			.navigate()
+			.waitForElementVisible('@homeScreen')
+			.click('@accessMenu')
+			.waitForElementVisible('@listScreen')
+			.navigate()
+			.waitForElementVisible('@homeScreen')
+			.click('@fieldsMenu')
+			.waitForElementVisible('@listScreen');
 	},
 	'Home view should allow clicking a card list item such as Users to should show the list of those items': function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.homeView.usersTabUnderDashboardAccessSubheading)
-			.waitForElementVisible(adminUI.cssSelector.listView.id)
-			.pause(browser.globals.defaultPauseTimeout);
+		browser.app
+			.navigate()
+			.waitForElementVisible('@homeScreen');
+
+		browser.homeScreen.section.accessGroup.section.users
+			.click('@label');
+
+		browser.app
+			.waitForElementVisible('@listScreen');
 	},
 	'Home view should allow an admin to create a new list item such as a user': function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.homeView.plusIconLinkForUsersTabUnderDashboardAccessSubheading)
-			.waitForElementVisible(adminUI.cssSelector.initialModalView.id)
-			.pause(browser.globals.defaultPauseTimeout);
+		browser.app
+			.navigate()
+			.waitForElementVisible('@homeScreen');
```

Comment:
ditto--the before function runs before each one of these, right?  Or does it run before the test file starts executing?